e2e/cli-e2e/tests/__snapshots__/help.e2e.test.ts.snap

```diff
@@ -3,59 +3,69 @@
 exports[`CLI help > should print help with help command 1`] = `
 "[1mCode PushUp CLI[22m
 
-Commands:
-  code-pushup               Shortcut for running collect followed by upload
+[32mCommands:[39m
```

Comment:
The colour codes make this snapshot unreadable, please remove them like how it's done in [report stdout tests](https://github.com/code-pushup/cli/blob/main/packages/utils/src/lib/reports/log-stdout-summary.integration.test.ts#L9-L12).

Reply:
done + replaced duplicate implementation

Reply:
Cool, thanks for extracting it :+1: